spec/presenters/blacklight/show_presenter_spec.rb

```diff
@@ -68,12 +64,10 @@ def mock_document_app_helper_url *args
         tmp_value = Capybara.ignore_hidden_elements
         Capybara.ignore_hidden_elements = false
         document.export_formats.each_pair do |format, _spec|
-          expect(subject).to have_selector("link[href$='.#{format}']") do |matches|
```

Comment:
Capybara evaluates blocks differently as of https://github.com/teamcapybara/capybara/issues/2616

Reply:
@sandbergja I don't understand. That ticket was closed with no action taken.

Reply:
Ah, I see it's https://github.com/teamcapybara/capybara/pull/2617#issuecomment-1368291831 where the change was made.